src/main/java/dk/kb/cfutv/service/CfuTvService.java

```diff
@@ -266,52 +261,47 @@ private String getYouSeeChannelId(String sBChannelId, Date date) throws ServiceE
         }
     }
 
-    private Date convertToUTC(Date date){
-        Locale locale = Locale.getDefault();
-        DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.DEFAULT,locale);
+    private ZonedDateTime convertToUTC(ZonedDateTime date){
+        return date.withZoneSameInstant(ZoneId.of("UTC"));
 
-        formatter.setTimeZone(TimeZone.getTimeZone("UTC"));
-        String utctime = formatter.format(date);
-        Date utcDate = new Date(utctime);
-        return utcDate;
     }
 
     /**
      * Translates a date to a String that looks like part of the url needed to access the download web page.
      * @param date to be translated.
      * @return String that looks like part of the url needed to access the download web page.
      */
-    private String dateToUrlPart(Date date){
+    private String dateToUrlPart(ZonedDateTime date){
         String result = "";
         //Year
         int year = date.getYear() + 1900; //Adjust for date.getYears() getting years since 1900.
         result += year;
         //Month
-        int month = date.getMonth() + 1; //Adjusting for date.getMonth() starting with 0 instead of 1.
+        int month = date.getMonthValue() + 1; //Adjusting for date.getMonth() starting with 0 instead of 1.
```

Comment:
ZonedDateTIme should not zero-index months, that was a thing in the old java.util.date class. See: https://docs.oracle.com/javase/8/docs/api/java/time/ZonedDateTime.html#getMonthValue--